.rubocop.yml

```diff
@@ -11,6 +11,7 @@ Govuk/GovukSubmit:
   Enabled: false
 
 AllCops:
+  TargetRubyVersion: 3.1.0
```

Comment:
What's this for? Do we need it, doesn't rubocop get the ruby version from `gemfile.lock`?

Reply:
You'd think that but the CI has other ideas. I'd prefer to be explicit so it doesn't complain

<img width="724" alt="Screenshot 2022-08-08 at 09 46 59" src="https://user-images.githubusercontent.com/616080/183378171-6f1c82cd-d5aa-4a62-9ad0-806dd05b0ad2.png">